apps/docs/src/components/Textarea.jsx

```diff
@@ -0,0 +1,53 @@
+import clsx from 'clsx'
+import {  useEffect, useRef } from 'react'
+
+// export interface Props {
+//   value: string
+//   className?: string
+//   placeholder?: string
+//   onChange: (e: ChangeEvent<HTMLTextAreaElement>) => any
+//   isOpen: boolean
+// }
+
+function Textarea({ value = '', className, placeholder, onChange, isOpen }) {
+  const ref = useRef(null)
+
+  useEffect(function handleFocus() {
+    if (!ref) return
+    if (isOpen) {
+      ref.current?.focus()
+    }
+  }, [isOpen])
+
+  return (
+    <textarea
+      ref={ref}
+      placeholder={placeholder}
+      value={value}
+      autoCapitalize="off"
+      autoCorrect="on"
+      autoComplete="off"
+      className={clsx(
+        'w-full',
+        'p-4',
```

Comment:
The space between the text and top looks too big to me. This is probably subjective, so feel free to ignore
```suggestion
        'py-3',
        'px-4',
```